src/controller/controller.ts

```
@@ -219,6 +219,14 @@ function shouldSpawn(room: Room, config: SpawnConfig): boolean {
             let count = Math.min(4, Math.ceil(ruinTombResourceCount / 450));
             config.maxCnt = count;
             return count > 0;
+        case ROLE_UPGRADER:
+            let storage = room.find(FIND_MY_STRUCTURES, {
+                filter: it => it.structureType === STRUCTURE_STORAGE
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)